pkg/nuclide-external-interfaces/1.0/atom.js

```diff
@@ -365,6 +365,7 @@ declare class atom$Pane {
   moveItemToPane(item: Object, pane: atom$Pane, index: number): void,
   destroyItem(item: Object): boolean,
   itemForURI(uri: string): Object,
+  getActiveEditor(): ?atom$TextEditor,
```

Comment:
This isn't a documented API. We try to limit the number of private APIs we include here. Can you explain why you need it, and why you can't make do with the public APIs?